test/test_license.py

```diff
@@ -26,5 +30,20 @@ def test_license_in_wheel(self):
                 '"python third_party/build_bundled.py" to regenerate it')
 
 
+    def test_distinfo_license(self):
+        """If run when pytorch is installed via a wheel, the license will be in
+        site-package/torch-*dist-info/LICENSE. Make sure it contains the third
+        party bundle of licenses"""
+
+        site_packages = os.path.dirname(os.path.dirname(torch.__file__))
```

Comment:
This dir would be `test/`, and tests aren't installed to begin with, so this cannot work?

Reply:
The test is intended for the CI runs which used the wheel to install. Do they install to `test/` rather than `site-packages` when doing `pip/conda install` ?

Reply:
If "they" is tests - they don't get installed at all. Note the directory layout:
```
torch/
test/
``
not `torch/test/`. There is nowhere for the tests to go. In CI, tests live in the cloned repo, while `torch/` can get installed to `site-packages`.

Reply:
Oh wait, I may be confused. Let me re-read the code.

Reply:
Ah okay, sorry my comment was confusing.

This test should indeed work, but not always. There's multiple ways to install, and `site_packages/torch-*dist-info/` is not guaranteed to exist, so I think you should skip the test if it doesn't.

Reply:
One more thing: In order for CI to pick up your test file you need to add it to the list of `TESTS = [ ... ` in test/run_test.py
currently it is not running on CI i am afraid.

Reply:
@walterddr Should we add a CI check that warns on files in `test` that aren't actually being run? Perhaps add another list in `test/run_test.py` that includes all the files that aren't tested, to ensure that any new file in `test` gets added to one of those lists?

Reply:
refactored the test to skip instead of silently pass, and added the file to `test/run_test.py`

Reply:
@samestep  good point we can follow up